File: 0070-climbing-stairs/0070-climbing-stairs.cpp
class Solution {
public:
    int climbStairs(int n) {
vector<int>dp(n+2 , 0);
dp[0]=0;
dp[1]=1;
if(n<=1) return n;
for(int i=2 ;i<=n+1  ;i++)
{
    dp[i]=dp[i-1]+dp[i-2];
}
        return dp[n+1];
    }
};

File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
List<Integer>dp=new ArrayList<>();
dp.add(0);
dp.add(1);
if(n<=1) dp.get(n);
for(int i=2; i<= n+1 ; i++)
{
    dp.add(dp.get(i-1)+dp.get(i-2));
}
// for(int i=0 ; i<dp.size() ; i++) System.out.println(dp.get(i));
return dp.get(n+1);
    }
}